From 572679c8e722c469a8b78580de9c60945c55ad47 Mon Sep 17 00:00:00 2001
From: Derek Poon <derekp@ece.ubc.ca>
Date: Wed, 5 Aug 2015 21:58:27 +0000
Subject: [PATCH] Don't detect devices named like c0d0p0 as disks

Devices named like c0d0 are disks; devices named like c0d0p0 are partitions,
and fai-disk-info should not report them as disks.
---
 lib/fai-disk-info |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/lib/fai-disk-info b/lib/fai-disk-info
index e1cbe00..82c6666 100755
--- a/lib/fai-disk-info
+++ b/lib/fai-disk-info
@@ -19,4 +19,4 @@ checkdisk() {
 }
 # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 # echo a space separated list of devices and their block size
-egrep ' etherd/e[[:digit:]]+\.[[:digit:]]+\b| i2o/hd.+\b| cciss/c.+d.+\b| ida/c.+d.+\b| rd/c.+d.+\b| fio.\b| hd.\b| sd[a-z]{1,2}\b|/disc\b| vd.\b| xvd.\b' /proc/partitions | checkdisk
+egrep ' etherd/e[[:digit:]]+\.[[:digit:]]+\b| i2o/hd.+\b| cciss/c[[:digit:]]+d[[:digit:]]+\b| ida/c[[:digit:]]+d[[:digit:]]+\b| rd/c[[:digit:]]+d[[:digit:]]+\b| fio.\b| hd.\b| sd[a-z]{1,2}\b|/disc\b| vd.\b| xvd.\b' /proc/partitions | checkdisk
-- 
1.7.2.5

